content/2-how-crs-works/2-3-false-positives-and-tuning.md

```diff
@@ -353,7 +353,7 @@ CRS ships with prebuilt *rule exclusion packages* for a selection of popular web
 
 The packages should be viewed as a good *starting point* from which to build upon. Some false positives may still occur, for example if working at a high paranoia level, if using a very new or old version of the application, if using plug-ins, add-ons, or user customizations.
 
-If using a native CRS installation, rule exclusion packages can be downloaded as [plugins]({{% ref "plugins.md" %}}).
+If using a native CRS installation, rule exclusion packages can be downloaded as [plugins]({{% ref "4-1-plugins.md" %}}).
```

Comment:
Shouldn't this link have the "new" form `< ref "4-about-plugins/4-1-plugins.md" >`?

Reply:
I think you don't even need to add subdirectories in `ref`, as hugo will find things properly.